web/themes/custom/novel/templates/fields/field--field-categories.html.twig

```diff
@@ -0,0 +1,27 @@
+{{ attach_library('novel/show-more') }}
+{% set show_more_text = '...'|t %}
+{% set show_less_text = 'Show less'|t %}
+
+{% if items|length > 1 %}
```

Comment:
I get why you're doing it, but instead of having an ìf`/`else`, why not just always print the parent, and then maybe add a `if item|length > 1, add class that JS listens for`